component/attr_cache/attr_cache.go

```diff
@@ -796,21 +796,21 @@ func (ac *AttrCache) GetAttr(options internal.GetAttrOptions) (*internal.ObjAttr
 	return pathAttr, getErr
 }
 
-// CreateLink : Mark the link and target invalid
+// CreateLink : Mark the new link invalid
```

Comment:
Good catch!
I just fixed this.
I also couldn't resist, and fixed all the asserts in the test file to use receivers (`suite.assertBlah(path, etc)` instead of `assertBlah(suite, path, etc)`.